README.md

```diff
@@ -94,4 +94,4 @@ Optional features:
 
 ## Rust Version Compatibility
 
-This crate is tested with the latest stable version of Rust. We do not currently test against other, older versions of the Rust compiler.
+Datafusion crate is tested with the [minimum required stable Rust version](https://github.com/search?q=repo%3Aapache%2Farrow-datafusion+rust-version+language%3ATOML+path%3A%2F%5ECargo.toml%2F&type=code)
```

Comment:
This is a clever way to get the right correct place 👍 

![Screenshot 2024-01-30 at 3 11 16 PM](https://github.com/apache/arrow-datafusion/assets/490673/99a37c9e-9031-47cf-b6f4-fe23bc741857)